### DISCUSSION TITLE
Tile fan speed add optional light entity on the end of the button line

### DISCUSSION THREAD

**Discussion by NonaSuomy** [home-assistant/frontend#18481] [Dashboard]
Add an optional light entity on the end of the button line for tile fan speed.
![image](https://github.com/home-assistant/frontend/assets/1906575/96ee918a-bb2f-41a9-ac3e-926edd72c793)

This is the closest I got with CardMod but I think it was causing glitches doing this and the button would randomly load massive after hitting refresh.
![image](https://github.com/home-assistant/frontend/assets/1906575/501a1a56-ba8d-426d-bd7c-34029cb0ef28) 

Some way to remove the icon/text header and %

**Comment by ildar170975** [comment]
![17cfe53ef352f43cdffc1fce93aa5717c6a6bffc](https://github.com/home-assistant/frontend/assets/71872483/fee8e544-d28f-463f-a1b4-b1d69679fc65)

https://community.home-assistant.io/t/card-mod-add-css-styles-to-any-lovelace-card/120744/5281?u=ildar_gabdullin
